src/main/java/io/jenkins/plugins/opentelemetry/semconv/JenkinsOtelSemanticAttributes.java

```diff
@@ -92,6 +92,7 @@
 
     public static final String SPAN_ID = "SPAN_ID";
     public static final String TRACE_ID = "TRACE_ID";
+    public static final String OTEL_EXPORTER_OTLP_TIMEOUT = "OTEL_EXPORTER_OTLP_TIMEOUT";
```

Comment:
I'm not sure it's a semantic attribute, I would create a dedicated list of OpenTelemetry Environment variables referring to the OTel specification:

* https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md
* https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure

Note that I have the same need on https://github.com/elastic/opentelemetry-maven-extension/blob/opentelemetry-maven-extension-0.1.0-beta-2/src/main/java/co/elastic/maven/opentelemetry/OpenTelemetrySdkService.java